netcore/excludes-System.Runtime.Tests.rsp

```diff
@@ -40,12 +41,6 @@
 # while the test passes, xunit fails in the end if it's enabled
 -nomethod System.Tests.Types.VoidTests.IsByRef_Get_ReturnsExpected
 
-# crashes runtime (Assertion at method-to-ir.c:12303, condition `load_opcode != OP_LOADV_MEMBASE' not met)
--nomethod System.Text.Unicode.Tests.Utf16UtilityTests.GetIndexOfFirstInvalidUtf16Sequence_WithSmallValidBuffers
--nomethod System.Text.Unicode.Tests.Utf16UtilityTests.GetIndexOfFirstInvalidUtf16Sequence_WithSmallInvalidBuffers
--nomethod System.Text.Unicode.Tests.Utf8Tests.ToChars_ValidBuffers
--nomethod System.Text.Unicode.Tests.Utf8Tests.ToBytes_WithLargeValidBuffers
```

Comment:
When did this get fixed? I have Mono from few days ago and it was still failing on those tests.

Reply:
Not sure, btw I've updated corefx version (tests) and can't reproduce these failures anymore

Reply:
I'll retest it and report back. It should not hold up this PR though.